### PR TITLE
feat(home): introduce the Hero

### PR DESCRIPTION
**Pull request by im-klau**
## Name PR according scheme: type of changes (feat/fix/refactor etc.) + description
feat(home): introduce the Hero

## Jira issue

JIRA issue: [FSU-112](https://klau.atlassian.net/browse/FSU-112)

## Checklist before requesting a review

- [x] Checked which branch you would merge.
- [x] Checked if the code follows the style guidelines of this project (use eslint and prettier).
- [x] Performed a self-review of code (bugs, clean code rules, etc).
- [x] Checked if added variables have unique names.
- [x] Checked if added code/function is not a duplicate. 
- [x] Tests have been added if possible.
- [x] New and existing unit tests pass locally with my changes.

### CODE REVIEW

File: packages/frontend/app/templates/HomePage/components/Hero/Hero.styled.js [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+  ${({ theme }) => theme.breakpoints.tabletLg} {
+    margin-bottom: 0;
+  }
+`
+
+export const Content = styled.div`
+  max-width: 580px;
+  padding-top: 70px;
+  padding-left: 70px;
```
Comment (AmandaKr) [inline packages/frontend/app/templates/HomePage/components/Hero/Hero.styled.js:31]:
```suggestion
padding: 0 70px;
```

File: packages/frontend/app/templates/HomePage/components/Hero/Hero.styled.js [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+  ${({ theme }) => theme.breakpoints.tabletLg} {
+    padding-left: 0px;
+    padding-bottom: 260px;
```
Comment (AmandaKr) [inline packages/frontend/app/templates/HomePage/components/Hero/Hero.styled.js:35]: padding: 0 0 260px 0;

File: packages/frontend/app/templates/HomePage/components/Hero/Hero.styled.js (hunk 1 above)
Comment (im-klau) [reply packages/frontend/app/templates/HomePage/components/Hero/Hero.styled.js:31]: I don't know why but I feel like these shorthand properties aren't so clear and it's easier to make a mistake using them 😱 `padding: 0 70px;` is different than padding-top: 70px and padding-left: 70px :D

Comment (AmandaKr) [reply packages/frontend/app/templates/HomePage/components/Hero/Hero.styled.js:31]: Ohh, sorry. You're right, it's easy to make mistake :D

File: packages/frontend/app/templates/HomePage/components/Hero/Hero.jsx [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+              bgColor={theme.colors.neutrals[100]}
+            />
+          ))}
+        </Styled.Cards>
+      </Styled.Container>
+    </Styled.Hero>
+  )
+}
+
+export default Hero
```
Comment (nata-kab) [inline packages/frontend/app/templates/HomePage/components/Hero/Hero.jsx:83]: <iframe src="https://giphy.com/embed/6nuiJjOOQBBn2" width="480" height="275" frameBorder="0" class="giphy-embed" allowFullScreen></iframe><p><a href="https://giphy.com/gifs/party-how-i-met-your-mother-applause-6nuiJjOOQBBn2">via GIPHY</a></p>

**Comment by nata-kab** [comment]
One little thing left to do. There is additional empty container in tablet view 😉 🕵️

Screenshot 2023-03-23 at 22 05 02

Screenshot 2023-03-23 at 22 03 25

**Comment by im-klau** [comment]
Thank you @nata-kab! I haven't noticed the additional space there 🙈 Please, review this one more time, it should be fine now 🤞

**Review by nata-kab — APPROVED** [review]
It's beautiful 🤩  :party_blob

**Pull request merged**